bench/src/main.rs

```diff
@@ -167,11 +168,13 @@ async fn bench(
     transaction_size: TransactionSize,
 ) -> Metric {
     let map_of_txs: Arc<DashMap<Signature, TxSendData>> = Arc::new(DashMap::new());
+    let total_gross_send_time = Arc::new(OnceCell::new());
```

Comment:
Return the instance instead of using Onecell.

Reply:
how that? can you show me?